tests/ui/escape_analysis.stderr

```diff
@@ -1,22 +0,0 @@
-error: local variable doesn't need to be boxed here
```

Comment:
The issue with this test is fixed on the master branch. Please leave this file unchanged.

Reply:
@flip1995 Thanks for the review!! Ok!I will fixe that later

Reply:
@flip1995 I fixed it:)